tests/ui/attributes/rustc_skip_during_method_dispatch.stderr

```diff
@@ -0,0 +1,88 @@
+error: malformed `rustc_skip_during_method_dispatch` attribute input
+  --> $DIR/rustc_skip_during_method_dispatch.rs:3:1
+   |
+LL | #[rustc_skip_during_method_dispatch]
```

Comment:
Error on line 3 emitted twice. You should add an exception in rustc_builtin_attrs to no longer validate this attribute. That exception will be temporary until all attrs are migrated. Take a look at what I did for the inline parser (search for sym::inline in the compiler what will also show you it).